lib/modules/manager/pub/artifacts.ts

```diff
@@ -24,6 +24,9 @@ export async function updateArtifacts({
   if (is.emptyArray(updatedDeps) && !isLockFileMaintenance) {
     logger.debug('No updated pub deps - returning null');
     return null;
+  } else if (updatedDeps.length === 1 && updatedDeps[0].depName === 'flutter') {
+    logger.debug('Only updated flutter sdk - returning null');
```

Comment:
what should happen when only dart sdk is updated? should we skip too?

Reply:
Right now the dark sdk is not being extracted. I've only added logic to extract flutter sdk. I can do that in another PR to also extract it.

Reply:
ok, yes do it a follow up PR